index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>Car Lot</title>
+
+    <!-- Bootstrap -->
+    <link href="/lib/node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="styles/quiz.css">
+  </head>
+  <body>
+
+    <div class="container">
+    	<h1>Check out our cars!</h1>
+
+    			<nav class="navbar" id="topNavbar">
+    				<!-- <a class="navbar-brand" href="#"> -->
+    				<ul class="nav navbar-nav">
+
+    					<li class="nav-item"><img src="images/lotlogo.jpg"</a></li>
+    					<!-- <li class="nav-item active">
```

Comment:
Try to not leave commented code in your projects when you turn them in. Comments are okay, commented code, not so much. :)